### ISSUE TITLE
Non-bubbling events won't trigger, but are accepted.

### ISSUE THREAD

**Issue by HLSiira** [cferdinandi/events#7]
**Test case:** https://codepen.io/hlsiira/pen/mdeyvVN

Hey, I think I have come across a weird edge case that I can't quite nail down the issue. 

When using Events, some event types don't bubble to the window, like MouseLeave, or MouseEnter, which means they are never triggered using Events. In trying to fix the issue, I tried to write a function to test whether or not an event type would bubble or not, and if it doesn't, add the listener directly to the element instead of to the window.

However, in trying to fix the issue, I noticed that MouseLeave seems to bubble in Firefox, but not in Chrome; more like in Firefox, MouseLeave acts like MouseOut.

Originally I figured it was just a simple case of detecting if the event type bubbles or not, but due to the weird Firefox VS Chrome handling, I'm not really sure if that is the actual issue, or if it's something else.

Thanks for making an awesome helper library, it's made coding way easier!

**Comment by cferdinandi** [comment]
That's an odd one! I'm using `useCapture` to force these to work. Let me debug further, thanks!

**Comment by HLSiira** [comment]
Yeah, I read more about it in your articles and was really confused as to why it wasn't working. It's been a long time since I messed with it and ended up having to convert the event into a bubbling one and just test what kind of event it was.

[This is the code](https://github.com/Atheos/Atheos/blob/40fc0a54f8452cd4b9b4db98f7466b3b57e5909a/components/sidebars/init.js#L158) where it should be used, but instead I had to create the work around, although I haven't gone back in a long time and messed with it since I *fixed it*
